config-templates/shopify/customer.yaml

```diff
@@ -0,0 +1,55 @@
+data_source:
+  name: shopify_customer
+  owners:
+    - support@transformdata.io
+  sql_table: SHOPIFY.CUSTOMER ### Update to your customers schema.table
+  mutability:
+    type: full_mutation
+    type_params:
+      update_cron: 0 2 * * *
```

Comment:
Mutability is optional now, and in Metricflow we only really support full_mutation. Shall we remove this (and all other mutability entries)?

At the very least we should remove the update_cron entry, since that's a Transform feature.
```suggestion
```